.github/ISSUE_TEMPLATE/breakage-form.yml

```diff
@@ -0,0 +1,64 @@
+---
+name: Breakage Form
+description: Report breakage that requires a config mitigation
+title: "Broken Site: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Each exception should be documented. Please create an issue using this form before creating a new exception. The exception should then link back to the created issue.
+        Set the title to the domain of the broken site.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description of Breakage
+      description: What is broken and how it is broken? (Include screenshots if possible)
+      placeholder: Page is blank for 5s after loading
+    validations:
+      required: true
+  - type: textarea
+    id: breaking-requests
+    attributes:
+      label: Tracker Requests breaking the site
+      description: Requests that must be unblocked for the site to function
+      placeholder: |
+        1. https://example.com/script.js
+        2. …
+    validations:
+      required: true
+  - type: checkboxes
+    id: affected-platforms
+    attributes:
+      label: Affected Platforms
+      description: Which DuckDuckGo products are affected by this breakage?
+      options:
+        - label: Browser Extensions
+        - label: Android
+        - label: iOS
+        - label: macOS
+        - label: Windows
+  - type: textarea
+    id: other
+    attributes:
+      label: Other Details
+      description: Any other information that may be useful
+  - type: markdown
+    attributes:
+      value: |
+        ## Additional Information
+  - type: dropdown
```

Comment:
These display as big headings for me. Will that also be the case live or no?

Reply:
Are you referring to the labels of the fields or "Additional Information". I can't speak to the size in the live form but I can only control strings like "Additional Information"

Reply:
The "label" field

Reply:
I don't think we can control this.